pyriemann/datasets/sampling.py

```diff
@@ -42,6 +43,96 @@ def _pdf_r(r, sigma):
     return np.exp(partial_1 + partial_2)
 
 
+def _rejection_sampling_2D_gfunction_plus(sigma, r_sample):
+    """ auxiliary function used for the 2D rejection sampling
+    algorithm in the case where r is sampled with
+    the function g+.
+    Parameters
+    ----------
+    sigma : float
+        Dispersion of the Riemannian Gaussian distribution.
+    r_samples : ndarray, shape (1, n_dim)
+        Sample of the r parameters of the Riemannian Gaussian distribution.
+    Returns
+    -------
+    probability_of_acceptation : float
+    """
+    MU_A = np.array([sigma**2/2, -sigma**2/2])
```

Comment:
such capital letters are used for constants not for regular variables
```suggestion
    mu_a = np.array([sigma**2/2, -sigma**2/2])
```